presto-docs/src/main/sphinx/sandbox-readmes/readme-hdp.rst

```diff
@@ -5,34 +5,34 @@ Readme for HDP VM Sandbox
 This is a VirtualBox sandbox image that can be used to experiment with the Presto distribution.
 The credentials for the VM are:
 
-    |  user: **root**
+    |  user: **presto**
     |  password: **presto**
 
 The image is based on Hortonworks HDP 2.2.4 and contains following components:
```

Comment:
Seems better to list the Cloudera and HDP versions -- can you double check for me the versions of those two?